RTS_GMLC/timeseries_pointers.json

```diff
@@ -3,2080 +3,3120 @@
         "simulation": "DAY_AHEAD",
         "category": "Generator",
         "component_name": "122_HYDRO_1",
-        "label": "get_max_active_power",
+        "module": "InfrastructureSystems",
+        "type": "Deterministic",
+        "label": "max_active_power",
+        "scaling_factor_multiplier": "get_max_active_power",
+        "scaling_factor_multiplier_module": "PowerSystems",
         "scaling_factor": 52.49761899362675,
         "data_file": "../forecasts/RTS_GMLC_forecasts/gen/Hydro/DAY_AHEAD_hydro.csv"
     },
     {
         "simulation": "DAY_AHEAD",
         "category": "Generator",
         "component_name": "122_HYDRO_2",
-        "label": "get_max_active_power",
+        "module": "InfrastructureSystems",
+        "type": "Deterministic",
+        "label": "max_active_power",
+        "scaling_factor_multiplier": "get_max_active_power",
+        "scaling_factor_multiplier_module": "PowerSystems",
         "scaling_factor": 52.49761899362675,
```

Comment:
@claytonpbarrows The term `scaling_factor` now seems confusing to me.  Should it be `normalization_factor`?

Reply:
I like `normalization_factor`